Update Lesson2_String.cpp

richa-jain1996 opened the pull request:
Please suggest any changes. Thanks!

richa-jain1996 committed:
Update Lesson2_String.cpp

omusavi commented:
Since the PR only lets me comment on what has changed compared to master I will leave all my comments here referencing line numbers:
1. Line 18 - Thanks for writing out the brute force solution! I won't be reviewing that closely in this case but good to know you are going through and using TEBOW IT!
1. Line 75 - Inconsistent spacing - this can be something that gets confusing, even during an interview and especially as you write code for a company. Doing correct spacing and indenting correctly is something that will help differentiate your code from others so its good to do it diligently! I see this also with your if statements in your tests, indenting the returns there makes it obvious they are a subset of the if statement. Personally I always use braces even in one line statements just to make it very clear.
1. Line 81 - For really long strings, this could be a pretty inefficient process to go and clear the values every single time you get a duplicate. Is there any way to make it so you don't need to do that every time?

richa-jain1996 committed:
Update Lesson2_String.cpp

Thanks for the comment. I have applied proper indentation and modified the code also.

richa-jain1996 committed:
Update Lesson2_String.cpp
